Sprint_Tests/dmitriy_bohdanov.md

```diff
@@ -0,0 +1,23 @@
+Раньше он хорошо разбирался в верстке и работе на передовых технологиях на фронте. После небольшого перерыва в несколько лет и глубокой депрессии плюс прослушивания тяжелой музыки, он слабо разбирается в современных тенденциях и много пьет. Если бы он справился с депрессией после смерти жены и ребенка, у него бы сейчас не стоял вопрос поиска работы с релевантной зарплатой.
+
+He used to be good in the markup and  work with the front line technologies on the frontend. After a short break break of a few years and deep depression combined with listening of gridcore he has a little knowledge of modern technologies and drinks a lot. If he had dealt with the depression after his wife and child deaths, he wouldn't had a question of finding a job with a relevant salary right now.
+
+Будь у него цель и адекватные зарплатные ожидания он бы уже работал на удаленке где нибудь в штатах или на Бали с перспективой перебраться на Канарские острова и оформить испанское гражданство. Он с детства мечтал жениться на испанке и рано выйти на пенсию и разводить виноград на побережье Балеарского моря.
+
+If he had a goal and realistic saraly expectation, he would be working on the remote somewhere in the US or in the Bali with a perspective of moving to the Canary islands an getting a spanish citizenship. Since childhood he had been dreaming of marrying with the spanian and retiring early and groving grapes on the Balearic sea coast.
+
+Если не знаешь как работает кусок кода, написанный коллегой тебе придется покрыть его консоль логами и проследить всю логическую цепочку. Это трудоемкое занятие. Если бы у нас была документация к этой фиче или хотя бы тикет на реализацию фичи с описанием бизнес логики, мы бы вылили фикс в течении пары часов.
+
+If you don't know how this code works, written by you colleague, you will have to cover it with console log's and trace the logical chain. It's a time consuming job. If we had documentation for this feature or at a least ticket for this feature implementation with the business logic description, we would have created a fix within a few hours.
+
```

Comment:
If you don't know how this code works, written by you colleague - `If you don't know how this code, written by you colleague ,works` - it is better to order the actions like that and put clarifications next to the word they clarify.
console.log's - is a posessive case but in your case it is a general case `console.logs`
`Job` is more like occupation `unlike` work which is the process
we would have created a fix within a few hours. - are you talking about the past, if yes then check the first sentence.